configure.ac

```diff
@@ -2621,12 +2621,22 @@ fi
 LIB_SOCKET_NSL
 AX_HARDEN_CC_COMPILER_FLAGS
 
-# link to ws2_32 if on mingw
+# if mingw then link to ws2_32 for sockets
 case $host_os in
-    *mingw32)
+    mingw*)
         LDFLAGS="$LDFLAGS -lws2_32" ;;
 esac
 
+# add wolfSSL defines for shared/static
+if test "$enable_shared" = "yes"
+then
+    AC_DEFINE([WOLFSSL_SHARED], [1], [Define if shared library avialble.])
+fi
+if test "$enable_static" = "yes"
+then
+    AC_DEFINE([WOLFSSL_STATIC], [1], [Define if static library avialble.])
```

Comment:
Good point. The build of it was checked, just that it produced both a static and shared library, but not using the library. There could be visibility issues with use, will test it.